### DISCUSSION TITLE
TLS certificate is being provisioned. This may take up to 15 minutes to complete

### DISCUSSION THREAD

**Discussion by anjanesh** [community/community#81552] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I pointed my subdomain (A records) to 185.199.111.153,  185.199.110.153, 185.199.109.153, 185.199.108.153 yesterday.
And it says "TLS certificate is being provisioned. This may take up to 15 minutes to complete."
![image](https://github.com/community/community/assets/85973/cf140102-6ef2-4fa5-bd8b-89ad20b3afed)

**Comment by yoannchaudet** [comment]
This is a normal message. Since you masked the domain in question, I cannot go look at the state of the certificate provisioning. If you made your DNS changes recently, it can delay the certificate provisioning as DNS propagation can take some time.

Do you still have an issue?

**Comment by anjanesh** [comment]
My DNS is at my hosting company - I only pointed the IP addresses to the subdomain http://music-festival.anjanesh.com

**Reply by yoannchaudet** [reply]
The configuration looks perfect.

Give it 10 minutes (for CDN propagation) and you should see your certificate: https://music-festival.anjanesh.com!

I nudged it.

**Reply by anjanesh** [reply]
Thanks. I had this like this since December 23rd I think (for a tech interview) - how did it (https) work magically right now ?

**Reply by yoannchaudet** [reply]
When you go to the settings page, it does that too. Try again to reissue the cert as needed.
